src/o365/spo/commands/orgassetslibrary/orgassetslibrary-list.ts

```diff
@@ -0,0 +1,117 @@
+import {
+  ContextInfo, ClientSvcResponse, ClientSvcResponseContents
+} from '../../spo';
+import config from '../../../../config';
+import request from '../../../../request';
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption,
+  CommandError
+} from '../../../../Command';
+import SpoCommand from '../../../base/SpoCommand';
+import { OrgAssetsResponse, OrgAssets } from './OrgAssets';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+}
+
+class SpoOrgNewsSiteListCommand extends SpoCommand {
+  public get name(): string {
+    return `${commands.ORGASSETSLIBRARY_LIST}`;
+  }
+
+  public get description(): string {
+    return 'List all libraries that are assigned as asset library';
+  }
+
+  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: (err?: any) => void): void {
+    let spoAdminUrl: string = '';
+
+    this
+      .getSpoAdminUrl(cmd, this.debug)
+      .then((_spoAdminUrl: string): Promise<ContextInfo> => {
+        spoAdminUrl = _spoAdminUrl;
+
+        return this.getRequestDigest(spoAdminUrl);
+      })
+      .then((res: ContextInfo): Promise<string> => {
+        const requestOptions: any = {
+          url: `${spoAdminUrl}/_vti_bin/client.svc/ProcessQuery`,
+          headers: {
+            'X-RequestDigest': res.FormDigestValue
```

Comment:
Could you please double check if we need the digest? I always test if I can get results without the digest with just access token. In a case of success I exclude it from the command since it is one request less. 

I ran your command without the digest and seemed to return data from the requests.

Reply:
It seems to work, so I don't mind removing it. However a quick search showed that currently in dev branche all commands that use `url: ${spoAdminUrl}/_vti_bin/client.svc/ProcessQuery,` are still using the x-requestdigest. You sure you want me to yank it out :)?